lib/http/apps/wallet.js

```diff
@@ -15,37 +15,48 @@ const password = process.env.PASSWORD || "coinsence";
  * @apiGroup Wallet
  * @apiVersion 1.0.0
  * 
- * @apiParam {Number} accountId Accounts unique ID.
+ * @apiParam {String[]} accountsIds Accounts unique ID.
  * 
  * @apiUse PostWalletSuccess
  * 
- * @apiUse MissingAccountIdError
+ * @apiUse MissingAccountsIdsError
  * @apiUse ExistantWalletError
  * @apiUse InternalServerError
  */
 app.post(`/`, (req, res) => {
-  createEncryptedWallet(res.locals.accountId, password).then(wallet => {
-    if (res.locals.coinsenceSigner) {
-      res.locals.coinsenceSigner.sendTransaction({
-        to: wallet.address,
-        value: ethers.utils.parseEther('0.1')
-      });
-    }
-    res.status(201).json({
-      address: wallet.address,
-      mnemonic: wallet.mnemonic
-    });
-  }).catch(e => {
-    console.log(e);
-    if(e.errno == -17) {
-      res.status(409).json({
-        error: "account already has a wallet"
+  const accountsIds = req.body.accountsIds;
+  let result = [];
+
+  if (!accountsIds) {
+    res.status(400).json({error: 'accountIds is missing'});
+  } else {
+    for(let accountId in accountsIds) {
+      createEncryptedWallet(accountId, password).then(wallet => {
+        if (res.locals.coinsenceSigner) {
+          res.locals.coinsenceSigner.sendTransaction({
+            to: wallet.address,
+            value: ethers.utils.parseEther('0.1')
+          });
+        }
+        result.push({
+          address: wallet.address,
+          mnemonic: wallet.mnemonic
+        });
+      }).catch(e => {
+        console.log(e);
+        if(e.errno == -17) {
+          res.status(409).json({
```

Comment:
this should not cancel the request and the loop. 
in the case that the wallet already exists we can load it and return the address or ignore it.